iphone/Classes/TiUIiOSProxy.h

```diff
@@ -171,6 +171,11 @@
 @property(nonatomic,readonly) NSNumber* BLUR_EFFECT_STYLE_DARK;
 #endif
 
+#if IS_XCODE_9
+@property(nonatomic,readonly) NSNumber* LARGE_TITLE_DISPLAY_MODE_AUTOMATIC;
```

Comment:
Is this really needed here? I have observed that if you call 
Ti.UI.iOS.LARGE_TITLE_DISPLAY_MODE_ALWAYS, it will directly call 

MAKE_SYSTEM_PROP(LARGE_TITLE_DISPLAY_MODE_AUTOMATIC, UINavigationItemLargeTitleDisplayModeAutomatic);

even if you have not make it property in .h file.